bundle/Command/GenerateProjectCommand.php

```diff
@@ -153,11 +154,48 @@ protected function execute(InputInterface $input, OutputInterface $output): ?int
         // Various cleanups
         $this->cleanup();
 
+        $this->activateGitHooks();
+
         $this->writeSection(['You can now start using the site!']);
 
         return 0;
     }
 
+    private function activateGitHooks()
+    {
+        if (!class_exists(Config::class)) {
+            return;
+        }
+
+        /** @var \Symfony\Component\Filesystem\Filesystem $fileSystem */
+        $fileSystem = $this->getContainer()->get('filesystem');
+        $projectDir = $this->getContainer()->getParameter('kernel.project_dir');
+        $captainHookFiles = [
+            $projectDir . '/captainhook_disabled.json',
+            $projectDir . '/captainhook_enabled.json',
+        ];
+
+        if ($fileSystem->exists($captainHookFiles)) {
+            return;
+        }
+
+        if ($this->questionHelper->ask(
+                $this->input,
+                $this->output,
+                $this->getConfirmationQuestion(
+                    'Do you want to use default <comment>githooks</comment>?',
+                    false
+                )
+            )
+        ) {
+            $fileSystem->symlink($projectDir.'/captainhook_enabled.json', $projectDir.'/captainhook.json');
```

Comment:
Looks like you didn't run `php-cs-fixer` :)

Reply:
Both this one and above fixed in af1d5fced893324db1edc9f4bb2bbeab17292d25